File: src/Java_P/List/ArrayList_To_LinkedList/ArrayList/AverageMarksOfStudent.java
package Java_P.List.ArrayList_To_LinkedList.ArrayList;

import java.util.Scanner;

public class AverageMarksOfStudent {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the number of student: ");
        int store = 0;

        int n = sc.nextInt();
       int [] marks = new int[n]; // integer array

        for (int i=0; i<marks.length;i++){
            marks[i] = sc.nextInt();
        }

        for (int j=0;j<n;j++){
            store+= marks[j];
        }
        store/=n;
        System.out.println( "Average Mark:"+ store);

    }
}

File: src/Kotlin/List/ArrayList/AverageMarksofStudent.kt
package Kotlin.List.ArrayList

import java.util.Scanner

fun main() {
    val sc = Scanner(System.`in`)
    var store =0
    val A = ArrayList<Int>()
    print("Enter number of student: ")
    val st:Int = sc.nextInt()

    for (i in 0 until  st){
        val mark = sc.nextInt()
        A.add(mark)
    }

    for (j in A){
        store +=j
    }
        store /=st

    println(store)
}